alkira/resource_alkira_connector_group.go

```diff
@@ -0,0 +1,91 @@
+package alkira
+
+import (
+	"log"
+
+	"github.com/alkiranet/alkira-client-go/alkira"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceAlkiraGroupConnector() *schema.Resource {
+	return &schema.Resource{
+		Description: "Manage connector groups\n\n" +
+			"Connector Groups can contain one or many connectors across different segments. " +
+			"This grouping of connectors can be for policy enforcement purposes or " +
+			"for monitoring purposes within the network. It allows for easier policy " +
+			"assignment by assigning policies to the entire group at the same time " +
+			"instead of having to assign them individually.",
+		Create: resourceGroupConnector,
```

Comment:
Nit: our current convention is to name this as `resource<name>Create`

Reply:
I checked a few other places in our repo. I'm the one who isn't keeping with convention by adding Create at the end of the function name.